alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php

```diff
@@ -1150,6 +1150,10 @@ public function execute()
 		$this->deliveryProfileId = $this->getRequestParameter( "deliveryProfileId", null );
 		$this->deliveryAttributes->setDeliveryProfileId($this->deliveryProfileId);
 
+		$preventCaptions = $this->getRequestParameter("preventCaptions",null);
+		if($preventCaptions && $preventCaptions == "true")
+			$this->deliveryAttributes->setCaptionOptionByName("preventCaptions", $preventCaptions);
```

Comment:
No need to add attributes for this param on deliveryAttributes , we just use this flag to set the value on $config->preventCaptions.

Reply:
done.